tests/kernel/fp_sharing/testcase.yaml

```diff
@@ -4,7 +4,7 @@ tests:
     platform_whitelist: frdm_k64f
     slow: true
     tags: core
-    timeout: 600
+    timeout: 3600
```

Comment:
An hour? Really???????
Something else is broken....

Reply:
Could be, raised a bug to track the issue: #7033